pkg/virt-handler/container-disk/mount.go

```diff
@@ -186,19 +186,20 @@ func (m *mounter) setMountTargetRecord(vmi *v1.VirtualMachineInstance, record *v
 
 // Mount takes a vmi and mounts all container disks of the VMI, so that they are visible for the qemu process.
 // Additionally qcow2 images are validated if "verify" is true. The validation happens with rlimits set, to avoid DOS.
-func (m *mounter) Mount(vmi *v1.VirtualMachineInstance, verify bool) error {
+func (m *mounter) Mount(vmi *v1.VirtualMachineInstance, verify bool) (map[string]*containerdisk.DiskInfo, error) {
```

Comment:
Previously there was only one call with `verify` set to `false` in `vmUpdateHelperMigrationTarget`:

https://github.com/kubevirt/kubevirt/blob/476fd77b90075b665a2a74f915131e22ac013791/pkg/virt-handler/vm.go#L2465

I am not sure why the verification was skipped at that point. Even though it is verified on the source node, I assume the image will be re-pulled at the target using the tag. And the tag does not guarantee that it will be that same image.

Anyway, currently it seems there are no more calls to `Mount` that do not need verification. So in that case I can probably rename the method to `MountAndVerify`. Would that be fine?

Reply:
> Anyway, currently it seems there are no more calls to `Mount` that do not need verification. So in that case I can probably rename the method to `MountAndVerify`. Would that be fine?

Sounds good. :+1: